src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -410,13 +412,23 @@ export default {
 			this.abortSearch()
 		})
 
-		this.fetchConversations()
+		// Restore last fetched conversations from browser storage,
+		// before updated ones come from server
+		this.restoreConversations()
+
+		this.onFetchingConversationsLeader().then(() => {
+			this.isFetchingConversationsLeader = true
+			this.fetchConversations()
+		})
 	},
 
 	mounted() {
-		// Refreshes the conversations every 30 seconds
+		// TODO update process when '@nextcloud/event-bus' broadcasting between tabs is available
```

Comment:
**thought:** We can use the `BroadcastChannel` even without `@nextcloud/event-bus` support. It's quite an easy-to-use API.

I'm not sure if it will be better in performance. On one hand, it clones data and interacts with all the tabs. On the other hand, WebStorage is a sync (blocking) PI + spends some time on serialization/deserialization.

But it would remove set interval and updating only once per 30 sec in subtabs. 🤔